### PR TITLE
Added the new waypoints library

### PR DESCRIPTION
**Pull request by brandomeniconi**

### CODE REVIEW

File: files/waypoints/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Caleb Troughton"
+github = "https://github.com/imakewebthings/waypoints"
+homepage = "http://imakewebthings.com/waypoints"
+description = "Waypoints is a library that makes it easy to execute a function whenever you scroll to an element."
+mainfile = "noframework.waypoints.min.js"
```
Comment (jimaek) [inline files/waypoints/info.ini]: `waypoints.min.js`

File: files/waypoints/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "waypoints",
+  "repo": "imakewebthings/waypoints",
+  "files": {
+    "basePath": "lib",
+    "include": ["*.min.*", "*.map", "waypoints.debug.js", "shortcuts/*.min.*"]
```
Comment (jimaek) [inline files/waypoints/update.json]: 1. Remove `waypoints.debug.js`
2. `./shortcuts/*.min.*` is correct

**Comment by brandomeniconi** [comment]
fixed as suggested

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**